src/extension/src/azure/azure-auth/azureAuth.ts

```diff
@@ -46,16 +46,35 @@ export abstract class AzureAuth {
      * Will get called whenever a function that uses AzureAccount object is called
      */
     if (this.api === undefined) {
-      this.api = extensions.getExtension<AzureAccount>(
+      this.api = vscode.extensions.getExtension<AzureAccount>(
         "ms-vscode.azure-account"
       )!.exports;
     }
   }
 
+  public static async promptUsersToLogout() {
+    return await vscode.window
+      .showInformationMessage(
+        DialogMessages.logoutPrompt,
+        ...[DialogResponses.yes, DialogResponses.no]
```

Comment:
We spread array here? Is it different than just passing these 2 arguments without array? :)

Reply:
I am not entirely sure. This is the definition of the `showInformationMessage` function:
`export function showInformationMessage<T extends MessageItem>(message: string, ...items: T[]): Thenable<T | undefined>;`

I did it the same way it is done for reset pages.

Reply:
The definition is fine, but I think you can still pass arguments directly as Piotr suggested. In the function, the arguments after `message` will be pushed into an array `items`

Reply:
Yes, so it's the 'rest parameters', it basically says 'gimme as many parameters as you want and I'll make an array from them' (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters). In the function call we declare the array and spread it, so we're just passing 2 arguments (it's equivalent to `showInformationMessage(DialogMessages.logoutPrompt, DialogResponses.yes, DialogResponses.no]`).
But, if you say it's consistent with the rest of the pages, let it be ;)

Reply:
I see. It makes sense to me now. I completely agree we can do it that way and in fact I can change it at both places. But I feel, that the way it is being processed in the function, this is more intuitive as the options (`yes` and `no`) are passed into the function.
It is kind of weird in general practice to make an array and spread it right away (lol) but I think here it makes sense to show the purpose of those arguments to the reader.